cardano_node_tests/tests/test_governance.py

```diff
@@ -151,7 +155,17 @@ def test_create_and_answer_poll(
             == required_signer_vkey_hash
         ), "The Tx has unexpected required signers."
 
-        # TODO: check required signers on dbsync
+        # Check required signers on dbsync
+        # The signatures should be stored on extra_key_witness table
+        if configuration.HAS_DBSYNC:
```

Comment:
This belong to `check_tx` in `dbsync_utils`. We even already have `extra_key_witness` in `TxRecord`.